src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -215,6 +218,39 @@ protected void Initialize(StorageEnvironment environment, DocumentDatabase docum
             }
         }
 
+        private void LoadValues()
+        {
+            TransactionOperationContext context;
+            using (_contextPool.AllocateOperationContext(out context))
+            using (var tx = context.OpenWriteTransaction())
+            {
+                var statsTree = tx.InnerTransaction.CreateTree(Schema.StatsTree);
+
+                var priority = statsTree.Read(Schema.PrioritySlice);
+                if (priority == null)
+                    Priority = IndexingPriority.Normal;
+                else
+                    Priority = (IndexingPriority)priority.Reader.ReadLittleEndianInt32();
+
+                var lastQueryingTime = statsTree.Read(Schema.LastQueryingTimeSlice);
+                if (lastQueryingTime == null)
```

Comment:
Actually no, because we are only deleting the AutoIndexes and they are marked in both cases with SystemTime.UtcNow